.changeset/slow-planes-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': patch
```

Comment:
If this PR will introduce new features, I believe that makes this a `minor` release?

If this PR revises existing property name case, that makes it a `major` release.

https://semver.org/

Reply:
Changed, thank you! https://github.com/cloudfour/cloudfour.com-patterns/pull/1264/commits/82c294dbd697d9389f0f455a5021fd71b8e11a4e